src/utils/src/ExponentialBackoffRetryStrategy.c

```diff
@@ -232,6 +232,19 @@ STATUS getExponentialBackoffRetryStrategyWaitTime(PRetryStrategy pRetryStrategy,
     return retStatus;
 }
 
+STATUS getExponentialBackoffRetryCount(PRetryStrategy pRetryStrategy, PUINT32 retryCount) {
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
```

Comment:
Nit: add null check for `pRetryStrategy`

Reply:
Done